### ISSUE TITLE
Using WP_STATELESS_MEDIA_UPLOAD_CHUNK_SIZE fails uploads to GCS

### ISSUE THREAD

**Issue by pilouski-adeo** [udx/wp-stateless#649]
Dear wp-stateless developers

Since we've setup WP_STATELESS_MEDIA_UPLOAD_CHUNK_SIZE in our wp-config.php uploading files which size is above WP_STATELESS_MEDIA_UPLOAD_CHUNK_SIZE is sending an error :

```
[09-Sep-2022 12:50:11 UTC] PHP Fatal error:  Uncaught Error: Class 'Google\Google_Http_MediaFileUpload' not found in /site/wp-content/plugins/wp-stateless/lib/classes/class-gs-client.php:234
Stack trace:
#0 /site/wp-content/plugins/wp-stateless/lib/classes/class-utility.php(352): wpCloud\StatelessMedia\GS_Client->add_media(Array)
#1 /site/wp-includes/class-wp-hook.php(307): wpCloud\StatelessMedia\Utility::add_media(Array, 996)
#2 /site/wp-includes/plugin.php(191): WP_Hook->apply_filters(Array, Array)
#3 /site/wp-includes/post.php(6584): apply_filters('wp_update_attac...', Array, 996)
#4 /site/wp-admin/includes/media.php(430): wp_update_attachment_metadata(996, Array)
#5 /site/wp-admin/incl in /site/wp-content/plugins/wp-stateless/lib/classes/class-gs-client.php on line 234
```

WP_STATELESS_MEDIA_UPLOAD_CHUNK_SIZE was set to 20971520

WP_STATELESS_VERSION 3.2.2

**Comment by pilouski-adeo** [comment]
uploads are working fine until WP_STATELESS_VERSION 3.1.1 and not working starting at version 3.2 (Google apis update)

**Comment by planv** [comment]
Fixed.
Will be released in version `3.2.3`

**Comment by planv** [comment]
Released. Closing the ticket.
Please let us know if you have any further questions
